apps/editor/widget/app.jsx

```diff
@@ -161,19 +192,13 @@ return (
       </div>
       <div>
         <Widget
-          src="nui.sking.near/widget/Layout.Modal"
+          src="devs.near/widget/modal.layout"
```

Comment:
Why'd you decide to make another version of it?

Reply:
Once we wanna close modal trought javascript functions and not only directly trougth Dialog.Triggers, I've created a new version. I tested doing the same with the nui.sking modal, but couldn't achieve the functionallity I wanted to.